CrrExistingObjects/metadataUtils.js

```diff
@@ -25,7 +25,10 @@ function listObjectVersions(params, log, cb) {
         keyMarker: params.KeyMarker,
         versionIdMarker: params.VersionIdMarker,
     };
-
+    log.debug('listing object versions', {
+        method: 'metadataUtils.listObjectVersions',
```

Comment:
random question here and kinda unrelated, but is there anyway to get the current method or context from `this` that could be more dynamic? Or maybe automated by werelogs in some way? I need to learn more about this but I would hate for our logs to show the wrong method cause we renamed it but forgot to update our logs. Sort of like how comments get out of date.

Reply:
IMO, I don't think we can grab it from `this`.

Reply:
It would be cool to have something like `Reflections` a la `c#` to get method/function names in js.